.gitignore

```diff
@@ -1,127 +0,0 @@
-# Logs
```

Comment:
],
	"main": "./out/extension.js",
	"contributes": {},

Reply:
you can research on GitHub marketplace for extension for js